src/whylogs/app/session.py

```diff
@@ -149,14 +149,17 @@ def logger(
         Create a new logger or return an existing one for a given dataset name.
         If no dataset_name is specified, we default to project name
 
-        Parameters
-        ----------
-        args: _LoggerKey
-            The properties of the logger if they're anything but the defaults.
-        Returns
-        -------
-        ylog : whylogs.app.logger.Logger
-            whylogs logger
+        Args:
+            dataset_name: name of the dataset
```

Comment:
Is this the format we settle on the documentation ?

Reply:
Yeah. Seems that Google Format plays nicer with the markdown code generation